mfa/methods/code/verify.md

```diff
@@ -0,0 +1,156 @@
+{% method %}
+
+## Two-Factor Verify Code
+Two-Factor authentication with Bandwidth services
+
+### Request URL
+
+<code class="post">POST</code>`https://mfa.bandwidth.com/api/v1/accounts/{accountId}/code/verify`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter     | Description                |
+|:--------------|:---------------------------|
+| to            | To telephone number        |
+| from          | From telephone number      |
+| applicationId | Application Id             |
+| scope         | scope                      |
+| code          | The code sent to requester |
+
+### Response Attributes
+
+| Property | Description        |
+|:---------|:-------------------|
+| valid    | Was the code valid |
+
+
+{% common %}
+
+### Example 1 of 1: Verify Code
```

Comment:
Add example 2, what an invalid verification looks like

Reply:
I'll add another response example with `valid: false`. I don't think another complete example block is necessary since it's just a true/false switch of the response